va/tlsalpn.go

```diff
@@ -198,6 +199,30 @@ func checkExpectedSAN(cert *x509.Certificate, name identifier.ACMEIdentifier) er
 	return nil
 }
 
+// Confirm that of the OIDs provided, all of them are in the provided list of
+// extensions. Also confirms that of the extensions provided that none are
+// repeated. Per RFC8737, allows unexpected extensions.
+func checkAcceptableExtensions(exts []pkix.Extension, requiredOIDs []asn1.ObjectIdentifier) error {
```

Comment:
(nit)

What about the `exclusive` operation model for this? e.g. We definitely don't want anything beyond `requiredOIDs`.

Not applicable in most places, but could be nice for tls-alpn.

Reply:
NVM this was addressed in a comment.  https://github.com/letsencrypt/boulder/pull/5919#issuecomment-1026146654

Reply:
I think that's too dangerous, see above.